### PR TITLE
Start markdown

### PR DESCRIPTION
**Pull request by ParserPixy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,50 +14,21 @@ _Organize ideas and collaborate using Markdown, a lightweight language for text
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
```
Comment (ParserPixy) [inline README.md:17]: hi

File: README.md [hunk 2]
```diff
@@ -14,50 +14,21 @@ _Organize ideas and collaborate using Markdown, a lightweight language for text
...
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 5: Merge your pull request
```
Comment (ParserPixy) [inline README.md:22]: Nice catch

File: README.md [hunk 3]
```diff
@@ -14,50 +14,21 @@ _Organize ideas and collaborate using Markdown, a lightweight language for text
...
-
-1. Right-click **Start course** and open the link in a new tab.
-2. In the new tab, most of the prompts will automatically fill in for you.
-   - For owner, choose your personal account or an organization to host the repository.
-   - We recommend creating a public repository, as private repositories will [use Actions minutes](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions).
-   - Scroll down and click the **Create repository** button at the bottom of the form.
-3. After your new repository is created, wait about 20 seconds, then refresh the page. Follow the step-by-step instructions in the new repository's README.
+1. Click **Merge pull request**.
+1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.
```
Comment (ParserPixy) [inline README.md:31]: Didnt think about this!

File: index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# This is an `<h1>` header, which is the largest
+![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)
+```
+$ git add .
+# Stage all files
+```
+- [x] Turn on GitHub Pages
+- [x] Outline my portfolio
+- [ ] Introduce myself to the world
```
Comment (ParserPixy) [inline index.md:9]: Now we're talking!

**Review by ParserPixy — COMMENTED** [review]
tldr: Good work overall!

**Comment by ParserPixy** [comment]
hi
